monkestation/code/datums/quirks/negative_quirks/corporate_sponsorship.dm

```diff
@@ -0,0 +1,16 @@
+/datum/quirk/corporate_sponsorship
+	name = "Corporate Sponsorship"
+	desc = "Become a living ad-break with a neural connection directly from the NT Advertisement Network to your mouth or an implanted speaker. Each ad is roughly 5 mins apart and has a revenue of 5 whole credits!"
+	value = -2
+	icon = FA_ICON_MONEY_BILL
+
+/datum/quirk/corporate_sponsorship/add()
+	var/datum/brain_trauma/mild/advert_force_speak/T = new()
+	var/mob/living/carbon/human/H = quirk_holder
```

Comment:
please don't use single-character var names. you could even do like `var/mob/living/carbon/human/quirk_holder = src.quirk_holder` here